test/test_manipulate_bifurcation.py

```diff
@@ -102,4 +102,8 @@ def test_bifurcation_angle(common_input, angle):
 
     if (angle_original - 2*common_input["angle"]) > np.pi:
         angle_rotated = 2*np.pi - angle_rotated
-    assert abs(angle_original - angle_rotated - 2*common_input["angle"]) < 0.08
+
+    if common_input["angle"] < 0:
+        assert abs(angle_original - angle_rotated - 2*common_input["angle"]) > 0.08
```

Comment:
This test is not valid.

On the left hand of `>` is the absolute difference between the measured new angle and the target. The test now just checks if the difference between target angle and actual change is larger than 0.08.

Please fix this before we merge this branch.